roles/container_runtime/tasks/extra_storage_setup.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Create file system on extra volume device
+  filesystem:
+    fstype: "{{ item.filesystem | default('xfs') }}"
+    dev: "{{ item.device }}"
+  when:
+    - item.format|default(false)|bool
+    - inventory_hostname in groups[item.hosts|default('all')]
```

Comment:
We only operate on masters and nodes now, no need for the inventory line.

Reply:
Actually, we need this to be able to constrain volumes to a subset of nodes and masters.

For example, I'd like to use this to mount etcd storage, and that can only live on masters.